masterlist.yaml

```diff
@@ -221,9 +221,12 @@ globals:
 groups:
   - name: &dlcGroup DLC
 
-  - name: &earlyLoadersGroup Early Loaders
+  - name: &eslmGroup Light Masters # Non DLC - Has Header Flags (ESM & ESL)
```

Comment:
I added a separate group as I think it's better to keep these separate from official esl DLC files.

Also as the behaviour of the two esl plug-in types is different.
I think it would be useful to refer to them differently.

> Light plug-in or `eslp` for .esp where only the ESL flag is set.
> Light Master or `eslm` for .esp or .esm with both ESL & ESM flags are set.